iceprog/Makefile

```diff
@@ -10,12 +22,19 @@ CFLAGS += $(shell for pkg in libftdi1 libftdi; do $(PKG_CONFIG) --silence-errors
 
 CFLAGS += -llibusb-1.0
 LDLIBS += -lusb-1.0
-endif
+endif #ifeq ($(STATIC),1)
+
+endif #ifeq ($(OS),macos)
 
 all: $(PROGRAM_PREFIX)iceprog$(EXE)
 
+ifeq ($(OS),macos)
```

Comment:
I tried with libusb from Homebrew and changed to

```
CFLAGS += -I /opt/homebrew/include
LDFLAGS += -L /opt/homebrew/lib -lusb-1.0
```

here and that seemed to work if I also changed the compile command to:

```
$(CC) $(CFLAGS) -o $@ $(LDFLAGS) $^ $(LDLIBS)
```

Note the inclusion of `CFLAGS`.

Preferable to support Homebrew, no?